setup.py

```diff
@@ -340,11 +338,45 @@ def run(self):
 
         if not has_errors:
             print('...PASSED')
+
+        all_playbooks, included_playbooks = find_playbooks()
+
+        print('#' * 60)
+        print('Invalid Playbook Include Checks')
+        invalid_include = []
+        for playbook in sorted(included_playbooks):
+            ignored = ['common', 'private', 'init', 'aws', 'gcp', 'openstack',
```

Comment:
We have entry points in common/openshift-cluster/upgrades/...

I'm not sure how this logic is working.  Could you add a comment?

Reply:
We should not have entry points in common/openshift-cluster/upgrades.  The entry points for those playbooks are in byo/openshift-cluster/upgrades.  I've added a verbose comment to the code to describe why each item is being ignored.